src/main/java/org/sciborgs1155/robot/Ports.java

```diff
@@ -1,6 +1,17 @@
 package org.sciborgs1155.robot;
 
+import java.util.HashMap;
+import java.util.Map;
+
 public final class Ports {
+
+  public static Map<Integer, String> idToName;
+
+  static {
+    idToName = new HashMap();
+    // idToName.put(id, name);
```

Comment:
Looks good! Please implement these names based on specific identifiers. The field names of the actual variables they are assigned to would be a good start.

For example, for the top shooter motor (6), you could name "SHOOTER_TOP_ROLLERS"

Try to keep things consistent. I know our naming is already inconsistent in order & content, but try and keep some constant thing going (like SUBSYSTEM_DESCRIPTOR_MECHANISM, or above)